src/test/java/com/jQueryTest/JQueryFirstAssignmentTest.java

```diff
@@ -1,7 +1,533 @@
+        boolean isNumericValueDisplayed = driver.findElement(By.xpath("//input[@name='value']")).isDisplayed();
+        Assert.assertTrue(isNumericValueDisplayed);
+        driver.switchTo().defaultContent();
+    }
+    @Test(priority = 16)
+    public void tabsTest(){
+        WebDriverWait wait = new WebDriverWait(driver,5);
+        WebElement clickOnTabsLink = driver.findElement(By.xpath("//a[text()='Tabs']"));
+        wait.until(ExpectedConditions.visibilityOf(clickOnTabsLink));
+        clickOnTabsLink.click();
+        boolean isTitleTabsDisplayed = driver.findElement(By.xpath("//h1[text()='Tabs']")).isDisplayed();
+        System.out.println("The title of Slider is displayed: " + isTitleTabsDisplayed);
+        WebElement iframe = driver.findElement(By.xpath("//iframe[@class='demo-frame']"));
+        wait.until(ExpectedConditions.visibilityOf(iframe));
+        driver.switchTo().frame(iframe);
+        WebElement swapToProinDolorTab = driver.findElement(By.xpath("//a[text()='Proin dolor']"));
+        wait.until(ExpectedConditions.visibilityOf(swapToProinDolorTab));
+        Actions actions = new Actions(driver);
+        actions.moveToElement(swapToProinDolorTab).click().release().build().perform();
+        boolean isProinDolorTabDisplayed = driver.findElement(By.xpath("//a[text()='Proin dolor']")).isDisplayed();
+        Assert.assertTrue(isProinDolorTabDisplayed);
+        driver.switchTo().defaultContent();
+    }
+    @Test(priority = 17)
+    public void tooltipTest(){
+        WebDriverWait wait = new WebDriverWait(driver,5);
+        WebElement clickOnTooltipLink = driver.findElement(By.xpath("//a[text()='Tooltip']"));
+        wait.until(ExpectedConditions.visibilityOf(clickOnTooltipLink));
+        clickOnTooltipLink.click();
+        boolean isTooltipDisplayed = driver.findElement(By.xpath("//h1[text()='Tooltip']")).isDisplayed();
+        System.out.println("The title of Slider is displayed: " + isTooltipDisplayed);
+        WebElement iframe = driver.findElement(By.xpath("//iframe[@class='demo-frame']"));
+        wait.until(ExpectedConditions.visibilityOf(iframe));
+        driver.switchTo().frame(iframe);
+        WebElement yourAge = driver.findElement(By.xpath("//input[@id='age']"));
+        wait.until(ExpectedConditions.visibilityOf(yourAge));
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("arguments[0].scrollIntoView();", yourAge);
+       Actions actions = new Actions(driver);
+       actions.moveToElement(yourAge).click().release().build().perform();
+       WebElement toolTipElement = driver.findElement(By.xpath("//input[@id='age']"));
+       String toolTipText = toolTipElement.getText();
+        System.out.println(toolTipText);
+        Assert.assertTrue(toolTipElement.getText().contains(toolTipText));
+        driver.switchTo().defaultContent();
+
```

Comment:
I recommend you to use "//" to leave comments just above the actions you're performing. So people can understand easily what you are trying to do there.